### PR TITLE
Added a way to proxy some paths to another port

### PR DESCRIPTION
**Pull request by wmira**
implemented #36 and added a way to proxy internally a list of path to another port

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,9 +46,10 @@ Command line parameters:
 * `--no-browser` - suppress automatic web browser launching
 * `--quiet` - suppress logging
 * `--open=PATH` - launch browser to PATH instead of server root
-* `--ignore=PATH` - comma-separated string of paths to ignore
+* `--ignore=PATH` - comma-separ	ated string of paths to ignore
```
Comment (tapio) [inline README.md]: Stray whitespace...

File: README.md [hunk 2]
```diff
@@ -46,9 +46,10 @@ Command line parameters:
...
 * `--entry-file=PATH` - serve this file in place of missing files (useful for single page apps)
 * `--wait=MILLISECONDS` - wait for all changes, before reloading
+* `--proxyPath=/path1,path2:redirectPort` - proxy internally path that starts with path1,path2 to redirect port
```
Comment (tapio) [inline README.md]: Existing cli flags use dash as word separator.

File: package.json [hunk 3]
```diff
@@ -1,38 +1,39 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tapio/live-server.git"
+  },
+  "engines": {
+    "node": ">=0.8.0"
+  },
+  "preferGlobal": true,
+  "license": "MIT"
 }
```
Comment (tapio) [inline package.json]: Please don't arbitrarily change indentation.

File: index.js [hunk 4]
```diff
@@ -14,7 +15,7 @@ var fs = require('fs'),
 var INJECTED_CODE = fs.readFileSync(__dirname + "/injected.html", "utf8");
 
 var LiveServer = {};
-
+var proxy = httpProxy.createProxyServer();
```
Comment (tapio) [inline index.js]: Would it be nicer not to create proxy at all if not requested in params?

File: package.json (hunk 3 above)
Comment (mgwalker) [inline package.json]: Just a note, months late, when you run an `npm` command, it often rewrites the `package.json` file itself and it indents with two spaces.  Likely this re-indentation was a result of npm rewriting the file.

Comment (tapio) [inline package.json]: Okay, good to know!

Comment (chimurai) [inline package.json]: @wmira 
Can you consider using https://github.com/chimurai/http-proxy-middleware
It is a tiny middleware written for connect/express.
The actual proxy functionality is provided by the same http-proxy lib.
The middleware eases proxy usage as middleware with some added sugar for example wildcard/glob support.

Let me know what you think.

Comment (wmira) [inline package.json]: @chumurai good stuff will try it out

**Comment by tapio** [comment]
Proxy support is now in master in another form through #111.

**Pull request closed without merging**